modules/installation-extend-edge-nodes-aws-local-zones.adoc

```diff
@@ -0,0 +1,115 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/cluster-tasks.adoc
+
+ifeval::["{context}" == "installing-aws-localzone"]
+:localzone:
+endif::[]
+
+:_content-type: PROCEDURE
+[id="installation-extend-edge-nodes-aws-local-zones_{context}"]
+= Creating user workloads in AWS Local Zones
+After you create an Amazon Web Service (AWS) Local Zone environment, and you deploy your cluster, you can use edge worker nodes to create user workloads in Local Zone subnets. 
+
+After the `openshift-installer` creates the cluster, the installation program automatically specifies a taint effect of `NoSchedule` to each edge worker node. This means that a scheduler does not add a new pod, or deployment, to a node if the pod does not match the specified tolerations for a taint. You can modify the taint for better control over how each node creates a workload in each Local Zone subnet.
+
+The `openshift-installer` creates the machine set manifests file with `node-role.kubernetes.io/edge` and `node-role.kubernetes.io/worker` labels applied to each edge worker node that is located in a Local Zone subnet.
+
+.Prerequisites
+
+* You have an access to the OpenShift CLI (`oc`).
+* You deployed your cluster in a Virtual Private Cloud (VPC) with defined Local Zone subnets.
+* You ensured that the machine set for the edge workers on Local Zone subnets specifies the taints for `node-role.kubernetes.io/edge`.
+
+.Procedure
+
+. Create a `deployment` resource YAML file for an example application to be deployed in the edge worker node that operates in a Local Zone subnet. Ensure that you specify the correct tolerations that match the taints for the edge worker node.
++
+.Example of a configured `deployment` resource for an edge worker node that operates in a Local Zone subnet
+[source,yaml]
+----
```

Comment:
Following @yunjiang29's [suggestion](https://github.com/openshift/openshift-docs/pull/57063/files#r1161551637) to add the Namespace to the bundle:

```suggestion
kind: Namespace
apiVersion: v1
metadata:
  name: <local_zone_application_namespace>
---
```

Reply:
Changed `kind: PersistentVolumeClaim` to `kind: namespace`

Reply:
@dfitzmau please revert this change, my suggestion was to add the `Namespace` fragment, not to replace. Please see the [discussion](https://github.com/openshift/openshift-docs/pull/57063/files#r1161551637) and [tested commands](https://github.com/openshift/openshift-docs/pull/57063#pullrequestreview-1373220967)

Reply:
Thanks, @mtulio . I reverted the change and updated the spec according to the [tested commands](https://github.com/openshift/openshift-docs/pull/57063#pullrequestreview-1373220967) link.